java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -190,6 +202,7 @@ open class DevToolsService : Service() {
     )
   }
 
+  // Remove once the client uses protos
```

Comment:
Here too?

Or actually, just here, and not in the places above, since if you delete this method you'll have to fix the callsites too